### PR TITLE
services: Update some function tables

### PR DESCRIPTION
**Pull request by lioncash**
Updated based on information from 3dbrew

### CODE REVIEW

**Comment by Subv** [comment]
LGTM

File: src/core/hle/service/ndm_u.cpp [hunk 1]
```diff
@@ -14,10 +14,26 @@ const Interface::FunctionInfo FunctionTable[] = {
...
     {0x00030000, nullptr,                 "QueryExclusiveMode"},
+    {0x00040002, nullptr,                 "LockState"},
+    {0x00050002, nullptr,                 "UnlockState"},
     {0x00060040, nullptr,                 "SuspendDaemons"},
+    {0x00070040, nullptr,                 "ResumeDaemons"},
     {0x00080040, nullptr,                 "DisableWifiUsage"},
     {0x00090000, nullptr,                 "EnableWifiUsage"},
+    {0x000A0000, nullptr,                 "GetCurrentState"},
+    {0x000B0000, nullptr,                 "GetTargetState"},
+    {0x000C0000, nullptr,                 "<Stubbed>"},
```
Comment (archshift) [inline src/core/hle/service/ndm_u.cpp]: ?

Comment (lioncash) [inline src/core/hle/service/ndm_u.cpp]: Yes

**Comment by bunnei** [comment]
LGTM

**Pull request merged**